src/test/resources/application.conf

```diff
@@ -7,7 +7,23 @@ akka {
       # enable DEBUG logging of actor lifecycle changes
       lifecycle = on
       unhandled = on
-      autoreceive = on
     }
   }
+
+  cluster {
```

Comment:
I think this should not be needed anymore. afaik only Scoop utilized Akka Cluster